libs/designsystem/reorder-list/src/reorder-list.component.scss

```
@@ -61,6 +61,14 @@ ion-backdrop {
   border-bottom: 20px transparent solid;
 }
 
+ion-reorder-group > div {
```

This needs to be qualified to target only `ion-reorder-group` that are direct descendants of the component, otherwise you also style sub-reorders :)
```suggestion
:host > ion-reorder-group > div {
```